tools/choco/tinytex/tinytex.nuspec

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>tinytex</id>
+    <version>0.25</version>
+    <packageSourceUrl>https://github.com/yihui/tinytex/tree/master/tools/choco/tinytex</packageSourceUrl>
+    <owners>naveen521kk</owners>
+    <title>TinyTex (Install)</title>
+    <authors>Naveen M K,Yihui Xie</authors>
+    <projectUrl>https://yihui.org/tinytex/</projectUrl>
+    <iconUrl>https://yihui.org/images/logo-tinytex.png</iconUrl>
+    <copyright>Copyright 2020 Naveen M K,Yihui Xie</copyright>
+    <licenseUrl>https://github.com/yihui/tinytex/blob/master/tools/choco/tinytex/LICENSE</licenseUrl>
```

Comment:
I think this should license should be pointed to Texlive license what to do you say @yihui ?

Reply:
Good question. I'm not sure...

Reply:
Ok then let it be.